docs/plugins/inputs/file.asciidoc

```diff
@@ -380,7 +380,7 @@ The sincedb record now has a last active timestamp associated with it.
 If no changes are detected in a tracked file in the last N days its sincedb
 tracking record expires and will not be persisted.
 This option helps protect against the inode recycling problem.
-Filebeat has a {filebeat-ref}/faq.html[FAQ about inode recycling].
+Filebeat has a {filebeat-ref}/faq.html#inode-reuse-issue[FAQ about inode recycling].
```

Comment:
REGRESSION:  The link target (#inode-reuse-issue) no longer exists after a reorg of the Beats FAQ.  I made this change to fix a broken doc build. We need to keep it.